### ISSUE TITLE
breakpoint not hit

### ISSUE THREAD

**Issue by jrieken** [Microsoft/vscode#1264]
```
function fn(...args: any[]) {
    console.log(args);
}

fn(1, 2, true, {});
```

* have the snippet above as **ts** with **sourcemaps**
* set breakpoints in the function body and on function invocation (last line)
* F5
*> only the breakpoint in the function body is hit, not the invocation breakpoint

**Comment by jrieken** [comment]
this is with node v5.2.0

**Comment by weinand** [comment]
This is a rare corner case: if "stopOnEntry" is false but you have a breakpoint in the same line where node would stop on entry, and if you are using sourcemaps, then it can happen that the breakpoint is skipped.

I've tried to improve this, but I'm sure that there are cases where this won't work.
@jrieken but with my fix your snippet works for me.

**Comment by jrieken** [comment]
It never stops for me on the last line, neither as described, nor with stop on entry, nor with setting the entry to app2_.ts_ instead of app2_.js_.

**Comment by weinand** [comment]
@jrieken I could repro the problem on node >= 0.12.9 and my fix works on all versions <= 5.1.1 but 5.2.0 breaks it.

**Comment by weinand** [comment]
@jrieken Since this fix is good for node <= 5.1.1 I do not want to revert it. Instead I created a new bug for addressing the issue with node 5.2.
